modular_skyrat/modules/rr_opt_in/code/objective.dm

```diff
@@ -0,0 +1,69 @@
+/datum/objective
+	/// The default opt in level of this objective. Only targets with opt in above or at this will be considered for this objective.
+	var/default_opt_in_level = RR_OPT_IN
```

Comment:
```suggestion
/datum/objective
	/// The default opt in level of this objective. Only targets with opt in above or at this will be considered for this objective.
	var/default_opt_in_level = RR_OPT_OUT
```